extension/README.md

```diff
@@ -225,49 +225,32 @@ When this command is  executed, VSCode...
 
 The same feature as above, but with debugging
 
-#### NAB: Edit Xliff Document (preview feature)
+#### NAB: Edit Xliff Document
 
 Opens XLF-files for editing in a webview.
 
 With the goal of reducing the clutter of XML files this feature is built for translators or non-developers in mind. Command available from right clicking a XLF-file and command palette. 
-
+![Edit Xliff Document](images/gifs/XliffEditorUsage.gif)
 ### Snippets
 
-#### Assign text variable with CopyStr
-
-Since CodeCop rule AA0139 complains on possible overflow, we need to assign text variables with a CopyStr statement
-
-#### Test Codeunit
-
-Inserts a stub Test Codeunit
-
-#### Test Function
-
-Inserts a stub Test Function
-
-#### Test SendNotificationHandler
-
-Inserts a generic SendNotificationHandler function
-
-#### Test MessageHandler
-
-Inserts a generic MessageHandler function
-
-#### Test ConfirmHandler
-
-Inserts a generic ConfirmHandler function
-
-#### Declare Dictionary
-
-Define variable of type Dictionary
-
-#### Declare List
-
-Define variable of type List
-
-#### Declare Enum value
-
-Declare enum value with caption.
+- Assign text variable with CopyStr
```

Comment:
@jwikman I felt that this would save some vertical space and make it more readable.